.circleci/config.yml

```diff
@@ -94,7 +97,7 @@ jobs:
           command: |
             pachctl port-forward &
             mkdir -p test-results
-            poetry run pytest -v --junitxml=test-results/junit.xml tests
+            DATUM_BATCHING_IMAGE=pachyderm/python-datum-batching-ci:$CIRCLE_SHA1 poetry run pytest -v --junitxml=test-results/junit.xml tests
```

Comment:
Can you store this `DATUM_BATCHING_IMAGE` in the environment at a higher level so that build, publish, and test can use it?

Reply:
We can, but just to check my understanding, is the env var used in those other jobs?